vault/blog.2021.2021-09-27-markdown-on-mobile.md

```diff
@@ -92,6 +119,32 @@ Here is a slew of other apps mentioned in the Dendron docs, and mentioned in the
 - [Textastic (iOS)](https://www.textasticapp.com/)
   - iPad + iPhone + Mac, paid, code-highlighting, ssh/FTP/SFTP/Dropbox/Google Drive support
 
+### Git clients
+
+- [MGit (Android)](https://manichord.com/projects/mgit.html)
+  - Git client, open source, free. There is a reason I repeatedly mention it in this article, when it comes to Android users, because it handles git repos well.
+- [Working Copy (iOS)](https://workingcopyapp.com/)
+  - Git client, paid (free version can't push commits)
+
+## Final thoughts
+
 Users may also be interested in future development around the [GitHub](https://github.com/mobile/) mobile client, which currently does _not_ support being able to edit or contribute new files. For now, people can use the app to post "LGTM" to PRs, add thumbs-down emojis to issues, and get notified when your PRs are rejected.
 
-[^1]: I know, using a laptop is a bit cramped when jammed between myself and the steering wheel. Sometimes, sacrifices in comfort are needed for a more productive life.
\ No newline at end of file
+`git` is not the only way to sync/backup your notes, since tools like Dropbox, Google Drive, iCloud, and Keybase can also help provide that functionality. I, and other Dendrologists, have a bias toward git due to a variety of reasons:
+
+- Following the [Docs as Code](https://www.writethedocs.org/guide/docs-as-code/) approach
+- Treating all documentation as part of a distributed wiki system that can import docs, from any source, into a single workspace
+- `git` versioning
+- Anyone that has used GitHub, GitLab, or other git-hosting solutions will have `git` installed
+
+If I had to choose what combinations to use, on Android, I'd recommend **Obsidian** with **MGit (Android)**. The two, together, have worked very well for me. I'm also a fan of setting up SSH keys to have project/repo-limited permissions as [Deploy Keys](https://docs.github.com/en/developers/overview/managing-deploy-keys#deploy-keys) on GitHub (with `Allow write access` enabled). This prevents users from setting up SSH keys with user-wide permissions across GitHub.
+
+---
+
+Interested in learning more about Dendron, for managing notes and documentation from your desktop?
```

Comment:
Can we use the following modification?

---

Enjoy this post? Subscribe to our newsletter!

<form
  action="https://buttondown.email/api/emails/embed-subscribe/dendron"
  method="post"
  target="popupwindow"
  onsubmit="window.open('https://buttondown.email/dendron', 'popupwindow')"
  class="embeddable-buttondown-form"
>
  <label for="bd-email">Enter your email</label>
  <input type="email" name="email" id="bd-email" />
  <input type="submit" value="Subscribe" />
  <p>
    <a href="https://buttondown.email" target="_blank">Powered by Buttondown.</a>
  </p>
</form>

Newsletters not your thing? You can also follow us elsewhere on the interwebs:
- Join [Dendron on Discord](https://discord.com/invite/xrKTUStHNZ)
- Follow [Dendron on Twitter](https://twitter.com/dendronhq)
- Checkout [Dendron on GitHub](https://github.com/dendronhq)

---

Interested in creating your own knowledge base using markdown, git, and VSCode? Get started with [Dendron](https://wiki.dendron.so/notes/678c77d9-ef2c-4537-97b5-64556d6337f1/) today.

Reply:
I just need the HTML necessary for embedding the subscribe box / HTML. If that can be provided here in a markdown HTML codeblock, I can copy/paste into the blog post where I currenlty am using an external link.